test/functional/test-friendly-iframe-embed.js

```diff
@@ -434,6 +434,18 @@ describe('friendly-iframe-embed', () => {
           + '<base href="https://acme.org/embed1">'
           + 'content');
     });
+
+    it('should insert CSP', () => {
```

Comment:
It seems like a better test here might be to try to include something bad (or, if you want to be thorough, all three kinds of bad things in separate tests) and assert that it doesn't load.

Reply:
Done